doc-content/enterprise-only/integration/sso-client-adapter-proc.adoc

```diff
@@ -72,15 +72,29 @@ In this example:
 The RH-SSO server converts the user names to lower case. Therefore, after integration with RH-SSO, your user name will appear in lower case in {PRODUCT}. If you have user names in upper case hard coded in business processes, the application might not be able to identify the upper case user.
 ====
 +
-. Navigate to the `_EAP_HOME_/standalone/configuration` directory in your {EAP} installation.
-  Locate the Elytron and undertow subsystem configurations in the `standalone.xml` and `standalone-full.xml` files and enable JACC. For example:
+. Elytron subsystem provides a built-in policy provider based on JACC specification. To enable the JACC manually in the `standalone.xml` or in the file where Elytron is installed, do any of the following tasks:
```

Comment:
Question - Can the below sentence be moved to an earlier section? If not, just add the article:

s/Elytron subsystem provides a built-in policy provider based on JACC specification./The Elytron subsystem provides a built-in policy provider based on the JACC specification.

Reply:
Done. Added an article.
That sentence should be present in the step itself.